### PR TITLE
[DOC-1434]Update Page Changelog

### PR DESCRIPTION
**Pull request by MarkWoulfeImprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Adding a missing page changelog to https://github.com/spatialos/UnrealGDK/pull/1532

#### Release note
NA

#### Tests
Serve Improbadoc ran

#### Documentation
NA

#### Primary reviewers
@ElleEss

### CODE REVIEW

File: SpatialGDK/Documentation/content/simulated-players.md [hunk 1]
```diff
@@ -122,5 +122,6 @@ Logs for simulated players are stored as [raw logs](https://docs.improbable.io/r
 
 <br/>
 <br/>------------<br/>
+_2019-11-28 Page updated without editorial review: added automatic sim players launching using batch scripts._<br/>
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/simulated-players.md]: "sim players" is not a term we use.
"automatic sim players launching" as a noun is quite hard to follow.


```suggestion
_2019-11-28 Page updated without editorial review: added batch scripts to launch simulated players._<br/>
```

**Comment by MarkWoulfeImprobable** [comment]
> A couple of bits.
> 
> Is there a fresh ticket to put this round again as it's published in draft. Should probs go in the 2010.1 "should" epic?

https://improbableio.atlassian.net/browse/DOC-1408 I have this ticket for it? Can add that to the epic.
